lib/shared/widgets/regiser_login_button.dart

```diff
@@ -0,0 +1,37 @@
+
+import 'package:flutter/material.dart';
+import 'package:flutter_screenutil/flutter_screenutil.dart';
+
+class LoginRegisterButton extends StatelessWidget {
```

Comment:
I believe we have a primary button? We can use the primary button but expose any property that may be different and just use it

Reply:
ok sir , i  done all the changes